src/main/java/org/commcare/cases/query/QuerySensitive.java

```diff
@@ -0,0 +1,9 @@
+package org.commcare.cases.query;
+
+/**
+ * Created by ctsims on 1/27/2017.
+ */
+
+public interface QuerySensitive {
```

Comment:
for reference we will likely implement indexed storage refactors soon that will use this interface as well